types/geojson/index.d.ts

```diff
@@ -1,123 +1,157 @@
-// Type definitions for GeoJSON Format Specification Revision 1.0
-// Project: http://geojson.org/
+// Type definitions for geojson 7946.0
+// Project: https://geojson.org/
 // Definitions by: Jacob Bruun <https://github.com/cobster>
 //                 Arne Schubert <https://github.com/atd-schubert>
+//                 Jeff Jacobson <https://github.com/JeffJacobson>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
+// Note: as of the RFC 7946 version of GeoJSON, Coordinate Reference Systems
+// are no longer supported. (See https://tools.ietf.org/html/rfc7946#appendix-B)}
+
 export as namespace GeoJSON;
 
-/***
- * http://geojson.org/geojson-spec.html#geojson-objects
+/**
+ * The valid values for the "type" property of GeoJSON geometry objects.
+ * https://tools.ietf.org/html/rfc7946#section-1.4
  */
-export interface GeoJsonObject {
-    type: string;
-    bbox?: number[];
-    crs?: CoordinateReferenceSystem;
-}
+export type GeoJsonGeometryTypeNames = "Point" | "LineString" | "MultiPoint" | "Polygon" | "MultiLineString" |
+    "MultiPolygon" | "GeometryCollection";
+
+/**
+ * The value values for the "type" property of GeoJSON Objects.
+ * https://tools.ietf.org/html/rfc7946#section-1.4
+ */
+export type GeoJsonTypeNames = "FeatureCollection" | "Feature" | GeoJsonGeometryTypeNames;
+
+/**
+ * Bounding box
+ * https://tools.ietf.org/html/rfc7946#section-5
+ */
+export type BoundingBox = [number, number, number, number] | [number, number, number, number, number];
```

Comment:
Could we simply name it `BBox` instead of `BoundingBox`.

Is anyone using Bounding Boxes with Z values?

Reply:
According to the [specification](https://tools.ietf.org/html/rfc7946#section-5) you have to set 4 numbers for 2D BBoxes and 6 numbers for 3D BBoxes instead of 5:

> Example of a 3D bbox member with a depth of 100 meters:
> {
> "type": "FeatureCollection",
>       "bbox": [100.0, 0.0, -100.0, 105.0, 1.0, 0.0],
>       "features": [
>       //...
>       ]
>   }